tests/test_utils_misc/__init__.py

```diff
@@ -109,11 +109,11 @@ def _test_with_crawler(mock, settings, crawler):
             else:
                 mock.assert_called_once_with(*args, **kwargs)
 
-        # Check usage of correct constructor using four mocks:
-        #   1. with no alternative constructors
-        #   2. with from_settings() constructor
-        #   3. with from_crawler() constructor
-        #   4. with from_settings() and from_crawler() constructor
+        # Check usage of correct __init__ method using four mocks:
+        #   1. with no alternative __init__ methods
+        #   2. with from_settings() __init__ method
+        #   3. with from_crawler() __init__ method
+        #   4. with from_settings() and from_crawler() __init__ method
```

Comment:
I think here "constructor" is replaced with `__init__` incorrectly; "constructor" here is a single word for from_crawler/from_settings/etc., not as a synonym to `__init__`.

Reply:
so should one leave it as is "constructor" or use “factory method” instead as @Gallaecio  mentioned in https://github.com/scrapy/scrapy/pull/4088#pullrequestreview-305189732?

Reply:
Yeah, I think so; no strong opinion about constructor/factory methods, I was always thinking about them as of "alternative constructors", so comment before the change as-is looks fine to me.

Reply:
@kmike thanks for the review, 7a84a4b is pushed.